Tomes/Tome.cs

```diff
@@ -3,16 +3,20 @@
 
 using Newtonsoft.Json.Linq;
 
-namespace Wizcorp.MageSDK.Tomes {
-	public class Tome {
+namespace Wizcorp.MageSDK.Tomes
+{
+	public class Tome
+	{
 		public delegate void OnChanged(JToken oldValue);
```

Comment:
in the future i would like to replace all these kind of delegates with 
'''
public event EventHandler<JToken> Changed;
'''

The subscribing class would then look like
'''
tome.Changed += (jToken, object) => {};
'''